package.json

```diff
@@ -3,6 +3,7 @@
     "type": "git",
     "url": "http://github.com/cfpb/cfgov-refresh.git"
   },
+  "license": "CC0-1.0",
```

Comment:
Scott and I discussed different ways of doing this in https://github.com/cfpb/cf-icons/pull/52